src/main/java/com/adorsys/ssi/sdjwt/SdJws.java

```diff
@@ -122,6 +123,34 @@ public void verifyNotBeforeClaim() throws SdJwtVerificationException {
         }
     }
 
+    /**
+     * Verifies that SD-JWT was issued by one of the provided issuers
+     * @param issuers List of trusted issuers to lowercase
+     */
+    public void verifyIssClaim(List<String> issuers) throws SdJwtVerificationException {
+        JsonNode issuer = payload.get("iss");
+
+        if (issuer == null) {
+            throw new SdJwtVerificationException("Missing 'iss' claim");
+        } else if (!issuers.contains(issuer.textValue().toLowerCase())) {
+            throw new SdJwtVerificationException("Unknown issuer: " + issuer.textValue());
+        }
```

Comment:
An `else` is not necessary when there is an early return. It even adds more cognitive stress.

```suggestion
        if (issuer == null) {
            throw new SdJwtVerificationException("Missing 'iss' claim");
        }
        
        if (!issuers.contains(issuer.textValue().toLowerCase())) {
            throw new SdJwtVerificationException("Unknown issuer: " + issuer.textValue());
        }
```